build.gradle

```diff
@@ -120,18 +120,19 @@ ext {
   lintApi = "com.android.tools.lint:lint-api:$lintVersion"
   lintTests = "com.android.tools.lint:lint-tests:$lintVersion"
 
-  composeVersion = "1.3.1"
-  composeUi = "androidx.compose.ui:ui:$composeVersion"
-  composeFoundation = "androidx.compose.foundation:foundation:$composeVersion"
-  composeRuntime = "androidx.compose.runtime:runtime:$composeVersion"
-  composeMaterial = "androidx.compose.material:material:$composeVersion"
-  composeUiToolingPreview = "androidx.compose.ui:ui-tooling-preview:$composeVersion"
-  composeTestJUnit = "androidx.compose.ui:ui-test-junit4:$composeVersion"
-  composeUiTooling = "androidx.compose.ui:ui-tooling:$composeVersion"
-  composeUiTestManifest = "androidx.compose.ui:ui-test-manifest:$composeVersion"
+  composeVersion = "2023.01.00"
+  composeBom = dependencies.platform("androidx.compose:compose-bom:$composeVersion")
```

Comment:
switched to the bom, so we can keep all the point releases in sync with the latest version, rather than having to define the version separately for each of the dependencies

Reply:
nice one!